Test/patterns/PatternMatching.dfy.expect

```diff
@@ -6,4 +6,4 @@ NestedVariableTest([2::3::4::5::6]) = 0, should return 0
 ConstantTest([3::4::5::6]) = 2, should return 2 
 M(P(11,10)) = 0, should return 0 
 M(P(-1,10)) = 3, should return 3 
-Testing MultipleNestedMatch: 012345, should return 012345
+Testing MultipleNestedMatch: 012345, should return 012345 
```

Comment:
If you can easily avoid having a space at the end of a line in the test output, then that would be good. Certain editors or commit filters may otherwise erase the space.